libbeat/processors/annotate/kubernetes/config.go

```diff
@@ -0,0 +1,30 @@
+package kubernetes
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	"time"
+)
+
+type kubeAnnotatorConfig struct {
+	InCluster             bool          `config:"in_cluster"`
+	KubeConfig            string        `config:"kube_config"`
+	Host                  string        `config:"host"`
+	Namespace             string        `config:"namespace"`
+	SyncPeriod            time.Duration `config:"sync_period"`
+	Indexers              PluginConfig  `config:"indexers"`
+	Matchers              PluginConfig  `config:"matchers"`
+	EnableDefaultMatchers bool          `config:"enable_default_matchers"`
+	EnableDefaultIndexers bool          `config:"enable_default_indexers"`
+}
+
+type PluginConfig []map[string]common.Config
+
+func defaultKuberentesAnnotatorConfig() kubeAnnotatorConfig {
+	return kubeAnnotatorConfig{
+		InCluster:             true,
+		SyncPeriod:            1 * time.Second,
+		Namespace:             "kube-system",
```

Comment:
i wouldnt recommend that. default namespace is only for control plane components. `kube-system` or `monitoring` are recommended namespaces.